### PR TITLE
Portfolio: Milestone 2

### PR DESCRIPTION
**Pull request by ambrose-kibet**
## In this milstone we:
- Add projects section
- Add about me section
- Add skills section

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -110,3 +110,185 @@ nav {
   padding: 0.01rem 8px;
   list-style-type: none;
 }
+
+.section-container {
+  background: #e5e5e5;
+  min-height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```
Comment (binhussen) [inline index.css]: - Kindly, use `diplay : grid` according to the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)

File: index.html [hunk 2]
```diff
@@ -64,6 +64,188 @@ <h2 class="info-title">I'am a Software Developer</h2>
...
+          <article class="languages">
+            <h3 class="generic-title">
+              <img src="./assets/skills.png" alt="square" /> Skills
+            </h3>
+            <p class="generic-info">Codekit</p>
+            <p class="generic-info">Github</p>
+            <p class="generic-info">Codepen</p>
+            <p class="generic-info">Gitlab</p>
+            <p class="generic-info">Terminal</p>
+          </article>
```
Comment (binhussen) [inline index.html]: - Kindly, make technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li` for a semantic reasons. 👍

File: index.html [hunk 3]
```diff
@@ -64,6 +64,188 @@ <h2 class="info-title">I'am a Software Developer</h2>
...
+        <div class="container">
+          <h1 class="projects-title">My Recent Works</h1>
+
+          <article class="project-card">
+            <div class="card-header">
+              <img
+                src="/assets/Img-Placeholder.png"
+                alt="multi-post stories"
+                class="card-header-img"
+              />
```
Comment (binhussen) [inline index.html]: - kindly, make sure your image url is typed correctly if the file is in the same directory use`./` not `/`. 👍 👍

File: index.css [hunk 4]
```diff
@@ -110,3 +110,185 @@ nav {
...
+  color: #5b5957;
+}
+
+.btn-block {
+  width: 100%;
+  text-align: center;
+}
+
+.project-card-variant {
+  background-image: url("/assets/card-bg.png");
```
Comment (binhussen) [inline index.css]: - kindly, make sure your image URL is typed correctly if the file is in some directory use`./` not `/`. 👍 👍

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @ambrose-kibet , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
- Kindly check all of your image URL
- Please check all of your requirements before doing it. like `display` property 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @ambrose-kibet 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
68747470733a2f2f6669636f666f72756d732e6d796669636f2e636f6d2f74352f696d6167652f736572766572706167652f696d6167652d69642f343932363069463839303835363833444139334135422f696d6167652d73697

## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. 🙏
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  Good commit history 🥇
 - [ ]  Good PR title/description 💯
 - [ ]  All linters passed 🥇
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**